docker/requirements.txt

```diff
@@ -0,0 +1,8 @@
+torch==0.4.1
```

Comment:
It would be better to use https://github.com/espnet/espnet/blob/master/tools/requirements.txt, then we can reflect the change of the main requirements.txt automatically. Also, why does it exclude cupy?

Reply:
cupy breaks the build for the cpu image so I tried to use a different requiremient. I will modify to exclude it using another command inside the container.